libs/cardano-ledger-core/src/Cardano/Ledger/UTxO.hs

```diff
@@ -218,6 +220,8 @@ class EraTxBody era => EraUTxO era where
     PParams era ->
     -- | Check whether a pool with a supplied PoolStakeId is already registered.
     (KeyHash 'StakePool (EraCrypto era) -> Bool) ->
+    -- | Function that can lookup current drep deposits
+    (Credential 'DRepRole (EraCrypto era) -> Maybe Coin) ->
```

Comment:
Produced should not have this argument. This should have been an indicator that something is wrong, cause I've mentioned this before that only one of the two functions needs a new argument.
```suggestion
```

Reply:
Sorry I didn't note that mention, and went down the wrong rabbit-hole on this one.